src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -1,24 +1,121 @@
 import theme from '../../../theme';
+import ContentLink from '../../components/ContentLink';
+import * as contentSelectors from '../../selectors';
 import { PopupBody } from '../../styles/PopupStyles';
+import * as pqSelectors from '../selectors';
+import IntroScreen from './IntroScreen';
+import ProgressBar from './ProgressBar';
 
 // tslint:disable-next-line:variable-name
 export const ShowPracticeQuestionsBody = styled(PopupBody)`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
   background: ${theme.color.neutral.darker};
+  padding: 2rem 3.2rem 2.5rem 3.2rem;
   ${theme.breakpoints.mobile(css`
     text-align: left;
     padding: 0;
   `)}
 `;
 
+// tslint:disable-next-line: variable-name
+export const SectionTitle = styled.h2`
+  font-size: 1.8rem;
+  line-height: 2.5rem;
+  color: ${theme.color.text.default};
+  margin-top: 0;
+  margin-bottom: 3rem;
+  ${theme.breakpoints.mobile(css`
+    font-size: 1.6rem;
+    margin: 1.4rem;
+  `)}
+`;
+
+// tslint:disable-next-line: variable-name
+export const QuestionsWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  border: 1px solid ${theme.color.neutral.darkest};
+`;
+
+// tslint:disable-next-line: variable-name
+export const QuestionsHeader = styled.div`
+  font-size: 1.4rem;
+  font-weight: bold;
+  color: ${theme.color.text.default};
+  background-color: ${theme.color.neutral.darkest};
+  height: 3.2rem;
+  width: 100%;
+  padding: 0 3.2rem;
+  display: flex;
+  align-items: center;
+`;
+
+// tslint:disable-next-line: variable-name
+export const StyledContentLink = styled(ContentLink)`
+  display: block;
+  font-size: 1.4rem;
+  color: #929292;
+  margin-top: 2.5rem;
+  text-decoration: none;
+
+  > span {
+    color: ${linkColor};
+
+    &::before {
+      content: " ";
+    }
+  }
+
+  ${theme.breakpoints.mobile(css`
+    margin: 1.2rem;
+  `)}
+`;
+
 // tslint:disable-next-line: variable-name
 const ShowPracticeQuestions = () => {
+  const book = useSelector(contentSelectors.book);
+  const section = useSelector(pqSelectors.selectedSection);
+  const questionsCount = useSelector(pqSelectors.questionsCount);
+  const currentQuestionIndex = useSelector(pqSelectors.currentQuestionIndex);
+  const selectedSectionHasPracticeQuestions = useSelector(pqSelectors.selectedSectionHasPracticeQuestions);
+
   return (
     <ShowPracticeQuestionsBody
       data-testid='show-practice-questions-body'
       data-analytics-region='PQ popup'
     >
+      {section ? <SectionTitle dangerouslySetInnerHTML={{ __html: section.title }} /> : null}
+      <QuestionsWrapper>
+        <QuestionsHeader>
+          <FormattedMessage id='i18n:practice-questions:popup:questions'>
+            {(msg: string) => msg}
+          </FormattedMessage>
+        </QuestionsHeader>
+        <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
+        {
+          selectedSectionHasPracticeQuestions && currentQuestionIndex === null
```

Comment:
can't this be `!currentQuestionIndex `?

Reply:
no, because `currentQuestionIndex` may be equal to `0` and `!0` is `true`